Microsoft.Toolkit/Microsoft.Toolkit.csproj

```diff
@@ -8,6 +8,7 @@
 
     <!-- This is a temporary workaround for https://github.com/dotnet/sdk/issues/955 -->
     <DebugType>Full</DebugType>
+    <Configurations>Debug;Release;OneDrive</Configurations>
```

Comment:
Same

Reply:
I'm not sure why this remained - I was trying to speed up my build by constraining it to just the OneDrive bits - I deleted the config so it shouldn't have been here - I'll stake it down and sprinkle it with some holy water this time.